src/app/components/room/room.service.js

```diff
@@ -144,12 +144,21 @@
             ActionService.enterRoom(msg.id, username, connection);
             // Step 3. Establish WebRTC connection with the Janus server
             // Step 4a (parallel with 4b). Publish our feed on server
+
+            var participants = 0;
             connection.publish({
+              success: function() {
+                  //start session unmuted?
+                  if(participants < 4){
```

Comment:
On the other hand, I think that you can get the list of participants from `msg` here, right? So you won't need that `participants` variable.